sdk/ai/Azure.AI.Projects.OneDP/src/Custom/Datasets/Datasets.cs

```diff
@@ -19,20 +19,20 @@ public partial class Datasets
         /// </summary>
         private (BlobContainerClient ContainerClient, string OutputVersion) CreateDatasetAndGetContainerClient(string name, string inputVersion)
         {
-            var pendingUploadResponse = StartPendingUpload(
+            var pendingUploadResponse = StartPendingUploadVersion(
                 name,
                 inputVersion,
                 new PendingUploadRequest(null, null, PendingUploadType.TemporaryBlobReference, null)
             );
 
             string outputVersion = inputVersion;
 
-            if (pendingUploadResponse.Value.BlobReferenceForConsumption == null ||
-                pendingUploadResponse.Value.BlobReferenceForConsumption.Credential?.Type != CredentialType.SAS ||
-                string.IsNullOrEmpty(pendingUploadResponse.Value.BlobReferenceForConsumption.BlobUri))
-            {
-                throw new InvalidOperationException("Invalid blob reference for consumption.");
-            }
+            //if (pendingUploadResponse.Value.BlobReferenceForConsumption == null ||
```

Comment:
Is commented code needed?

Reply:
We shall remove Agents custom code from https://github.com/Azure/azure-sdk-for-net/tree/feature/azure-ai-projects-1dp, which means you can remove all Agents custom code